src/heuristic/HeuristicMapper.cpp

```diff
@@ -138,6 +139,21 @@ void HeuristicMapper::map(const Configuration& configuration) {
     }
   }
 
+  if (results.heuristicBenchmark.expandedNodes > 0) {
+    results.heuristicBenchmark.timePerNode /=
+        static_cast<double>(results.heuristicBenchmark.expandedNodes);
+    results.heuristicBenchmark.averageBranchingFactor =
+        static_cast<double>(results.heuristicBenchmark.generatedNodes -
+                            layers.size()) /
+        static_cast<double>(results.heuristicBenchmark.expandedNodes);
+    for (const auto& layer : results.layerHeuristicBenchmark) {
+      results.heuristicBenchmark.effectiveBranchingFactor +=
+          layer.effectiveBranchingFactor *
+          (static_cast<double>(layer.expandedNodes) /
+           static_cast<double>(results.heuristicBenchmark.expandedNodes));
```

Comment:
I also might be mistaken on this, but I think the resulting precision might be higher this way. Since `expandedNodes` are usually rather large numbers (both in the global and the layer-wise case), I thought it might be beneficial to do that division before the multiplication. However, truthfully, I do not even know if the compiler adheres to the order of operations, that I tried to suggest via the parentheses.

Reply:
Although I am fairly sure that the compiler does not adhere to the order (something in the back of my mind tells me that we enable some performance optimization flag that eliminates this behavior), I find you reasoning convincing 👍🏻